cmd/utils/revert_encryption.go

```diff
@@ -0,0 +1,63 @@
+package utils
+
+import (
+	"errors"
+	"fmt"
+	"github.com/frain-dev/convoy/config"
+	"github.com/frain-dev/convoy/database/postgres"
+	"github.com/frain-dev/convoy/internal/pkg/cli"
+	"github.com/frain-dev/convoy/internal/pkg/keys"
+	"github.com/frain-dev/convoy/pkg/log"
+	"github.com/spf13/cobra"
+)
+
+func AddRevertEncryptionCommand(a *cli.App) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "revert-encryption <encryption-key>",
+		Short: "Reverts the encryption initialization for the specified table columns with the provided encryption key",
+		Args:  cobra.ExactArgs(1),
```

Comment:
Do we need to include this? Since we have already retrieved the key from line 20. Or is that how it's meant to work?

Reply:
It's a sanity check. To be sure they know what they are doing.

Reply:
Got it